homeassistant/components/kodi/media_player.py

```diff
@@ -717,6 +717,8 @@ async def async_play_media(
             await self._kodi.play_channel(int(media_id))
         elif media_type_lower == MEDIA_TYPE_PLAYLIST:
             await self._kodi.play_playlist(int(media_id))
+        elif media_type_lower == "file":
+            await self._kodi.play_file(str(media_id))
```

Comment:
No need to cast to string, it's already a string.
```suggestion
            await self._kodi.play_file(media_id)
```

Reply:
Thanks! That's what I get for blindly copy-pasting "code that worked" ;-)